src/app/components/account/account-list-item-layout.tsx

```diff
@@ -20,6 +22,12 @@ interface AccountListItemLayoutProps extends StackProps {
   balanceLabel: React.ReactNode;
   onSelectAccount(): void;
 }
+
+function AddressWrapper({ walletType, children }: { walletType: WalletType; children: ReactNode }) {
+  if (walletType == WalletType.Ledger) return <>{children}</>;
+  return <CaptionDotSeparator>{children}</CaptionDotSeparator>;
+}
```

Comment:
How does this look if you have both chain Ledger keys installed? The same with no separator?

I reckon the problem here is that CaptionDotSeparator doesn't work well with `children`. It shows `• bitcoin address` because internally it detects there are two items in the array. I wonder if there's a better way to pass addresses to it, without children. Maybe

```ts
<CaptionDotSeparator content={[stxAddress, btcAddress]} />
````

where internally it filters the array for undefined. If it has an array.length = 1, it shouldn't show a dot

Reply:
Agreed, I think that's probably a better approach. I'll remove this change and improve this section in a follow up, thanks!